src/app/Modules/auth/pages/forgot-password/forgot-password.component.md

```diff
@@ -0,0 +1,17 @@
+# Forgot Password Component
```

Comment:
I think when a Markdown file is the only one in its folder you should call it `README.md` which will cause it to be automatically parsed and displayed in Github page - same for other files here